test/Microsoft.TemplateEngine.TestHelper/TestBase.cs

```diff
@@ -22,7 +22,18 @@ protected TestBase()
             };
 
             EngineEnvironmentSettings = new EngineEnvironmentSettings(host, x => null);
-            host.VirtualizeDirectory(Environment.ExpandEnvironmentVariables("%USERPROFILE%/.templateengine"));
+            host.VirtualizeDirectory(GetTemplateEngineDirectory());
+        }
+        
+        private static string GetTemplateEngineDirectory()
+        {
+            string envValue = Environment.GetEnvironmentVariable("%USERPROFILE%");
+            if (!string.IsNullOrWhiteSpace(envValue))
+            {
+                return Environment.ExpandEnvironmentVariables("%USERPROFILE%/.templateengine");
+            }
+
+            return Environment.ExpandEnvironmentVariables("$HOME/.templateengine");
```

Comment:
These are not always set in nix boxes right? Like CI? We hit issues with NuGet because of this. Since this is a test class, would it be possible to set this to a folder under the repo, for instance?